index.html

```diff
@@ -14,8 +14,6 @@
         <nav class="navbar">
             <div class="logo">
               <a href="#">My Logo</a>
-            </div>
-            <div class="menu-icon">
```

Comment:
- [ ] Dear @jerymey, Your navbar navigation items are hidden from the screen for screen sizes from `768px` to `1400px`, Please make sure we can always see the navigation items on the screen and the layout stays the same. 
- [ ] Also the logo is in multi line text, This is different from the original design. Please take a look and make some changes.
### Screen size >768px 
![image](https://github.com/jerymey/Portfolio-setup/assets/69077061/fba791f3-c302-483b-a0cc-140110a41802)